components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityCoreConstants.java

```diff
@@ -90,8 +90,10 @@ public class IdentityCoreConstants {
     // Pagination constants.
     public static final int DEFAULT_MAXIMUM_ITEMS_PRE_PAGE = 100;
     public static final int DEFAULT_ITEMS_PRE_PAGE = 15;
+    public static final int USERS_LIST_PER_ROLE_LOWER_BOUND = 1000;
```

Comment:
shouldn't this be 100? In your description also you have mentioned 100

Reply:
The description is outdated. After an offline discussion, the [default value](https://github.com/wso2/carbon-identity-framework/pull/5878/files#diff-ccd8d78f18b752f00ba42b5384248d7b8e8c6cc965a44569e2654ca6ce5bb616R804) was raised to lower bound of 1000. We noticed that if the lower value was less than the default value of MaxUserNameListLength config set for a certain userstore, the UI would showcase an incomplete number of roles.

I will update the description to the correct value.